src/test.js

```diff
@@ -0,0 +1,21 @@
+import { add, subtract, multiply, divide } from "./index.js";
+
+test("2 + 5은 5입니다.", () => {
```

Comment:
describe, it으로 테스트 목적을 나누면 더 좋습니다.

```suggestion
describe("calculate add(a, b)", () => {
  ...
  it("2개의 숫자를 더한 값을 계산한다", () => {
    expect(add(2, 3)).toBe(5);
  }
})
```

Reply:
각 테스트 케이스별로 여러 경우의 테스트가 추가되면 좋을 것 같습니다.
3자리, 3자리의 연산이나 소숫점 연산, -연산도 들어가면 좋습니다